README.md

```diff
@@ -60,7 +63,35 @@ tox
 
 ## Runtime Environment
 
-Install (from sources) system-wide:
+### Install pre-requirements
+```shell
+sudo yum -y install epel-release
+sudo yum -y install \
+    libibverbs \
+    NetworkManager \
+    NetworkManager-libnm \
+    NetworkManager-ovs \
+    openvswitch \
+    python2-pyyaml \
+    python2-six \
+    python-gobject-base \
+    python-jsonschema \
+    python-setuptools \
+    python-ipaddress
+sudo echo -e "[device]\nmatch-device=*\nmanaged=0\n" >> \
+    /etc/NetworkManager/conf.d/97-docker-build.conf
+sudo echo -e "[main]\nno-auto-default=*\n" >> \
+    /etc/NetworkManager/conf.d/97-docker-build.conf
+```
```

Comment:
openvswitch needs to be started and NetworkManager needs to be (re)started as well.

Reply:
Right, thanks.

Reply:
Probably it makes sense to also enable it, therefore

```
systemctl enable --now NetworkManager openvswitch
systemctl restart NetworkManager
```

Is better - also NetworkManager might need to be restarted after starting openvswitch (not 100% sure).

Reply:
nmstate installation is not about deciding this for the user. I think NM is enabled by default when installed and regarding OVS, I do not know and prefer not to interfere.

Reply:
Documenting how it works best is not deciding this for the user.... You realize that these are steps that a user will do manually, so there is no inteference as long as the user does not do anything. But not telling them about the best options is strange.... Also making them add custom config files to NM is a lot worse that asking them to enable services.

Reply:
Done